app/views/buildings/show.html.erb

```diff
@@ -35,38 +31,39 @@
     </div>
   </div>
 <% end %>
-
-<div class="row">
-  <div class="col-md-11 col-md-offset-1">
```

Comment:
I didn't see a great benefit in indenting the label or the table, so I removed the above.